src/main/kotlin/org/web3j/evm/ConsoleDebugTracer.kt

```diff
@@ -631,7 +615,7 @@ open class ConsoleDebugTracer(protected val metaFile: File?, private val reader:
                 input.trim().toLowerCase() == "abort" -> {
                     val enumSet = EnumSet.allOf(ExceptionalHaltReason::class.java)
                     enumSet.add(ExceptionalHaltReason.NONE)
-                    throw ExceptionalHaltException(enumSet)
```

Comment:
if we are throwing directly the `ExceptionalHaltReason.NONE`, why do we need the `enumSet` ?